src/pages/EmployeeOverview/components/FilterEmployeeModal.jsx

```diff
@@ -0,0 +1,175 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  OutlinedInput,
+  Select,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import { useState } from "react";
+
+/* The code block is defining constants and an object that are used for styling the dropdown menus in
+the `FilterEmployeeModal` component. */
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
```

Comment:
Does it make sense to define this in theme data?

Reply:
No, the theme don't provide keys for this and i dont know which influence it has, to add custom one...